osp/wrappers/simcmclkinetics/kinetics_engine.py

```diff
@@ -89,6 +89,7 @@ def parseResults(self, jsonResults, root_cuds_object, synEntityToCUDSmap):
             else:
                 outputs_file = open("output_results.txt", "w")
                 for result_item in results:
+                    root_cuds_object.add(result_item, rel=CMCL.HAS_PART)
```

Comment:
I added this line because if a CUDS object is not reachable (no matter if directly or indirectly) from the Wrapper object (the root), then it is actually not possible for the user to retrieve the object without hacks.